_posts/2022-08-24-dc30-hhv-challenges.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: DEF CON 30 HHV CTF Recap
+date: 2022-08-24 12:00:00 -0000
+author: mediumrehr
+author_url: https://blog.rehr.io/
+image: assets/images/header.png
+categories: [CTF, DC30]
+---
+
+Thank you all for another great event! It was good to be back in person for DC30, and we're extremely pleased with the turnout for this year's CTF. Congratulations to elklepo for winning this year's HHV CTF. Challenge descriptions, write ups, and stats are now all posted on the [DC30 HHV Challenge Page](/challenges/dc30).
```

Comment:
Do we want to do this callout HaD style and put the handle in square brackets? Not super critical either way just wondering if it would make it stand out more?

Reply:
we haven't done that before, but doesn't mean we can't start now. i'm good either way. i'm not sure it stands out that much more with brackets though
<img width="923" alt="image" src="https://user-images.githubusercontent.com/3074997/186489080-de2ada29-7f55-476c-943c-bb2af21608b9.png">

Reply:
I honestly have no preference either way either, can probably just leave it alone. @ShortTie, @BenGardiner, any thoughts?

Reply:
Hmmm. Sorry to 'option C' but maybe `**@handle**` ?

e.g.

**@rehr** and **@kbembedded** shoutouts!

Reply:
i don't agree with adding the @, since we have something to link to, but i tried the bold. still no preference.

<img width="911" alt="image" src="https://user-images.githubusercontent.com/3074997/186513389-8ec743aa-4cc1-4de5-9bf4-d065476087c8.png">

Reply:
that's great too